diff --git a/node_modules/expo-audio/android/src/main/java/expo/modules/audio/AudioModule.kt b/node_modules/expo-audio/android/src/main/java/expo/modules/audio/AudioModule.kt
index 992239b..37acb39 100644
--- a/node_modules/expo-audio/android/src/main/java/expo/modules/audio/AudioModule.kt
+++ b/node_modules/expo-audio/android/src/main/java/expo/modules/audio/AudioModule.kt
@@ -148,7 +148,7 @@ class AudioModule : Module() {
     val result = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
       val requestType = interruptionMode?.let {
         if (it == InterruptionMode.DO_NOT_MIX) {
-          AudioManager.AUDIOFOCUS_GAIN_TRANSIENT
+          AudioManager.AUDIOFOCUS_GAIN
         } else {
           AudioManager.AUDIOFOCUS_GAIN_TRANSIENT_MAY_DUCK
         }
@@ -157,9 +157,11 @@ class AudioModule : Module() {
         setAudioAttributes(
           AudioAttributes.Builder()
             .setContentType(AudioAttributes.CONTENT_TYPE_MUSIC)
+            .setUsage(AudioAttributes.USAGE_MEDIA)
             .build()
         )
         setAcceptsDelayedFocusGain(true)
+        setWillPauseWhenDucked(interruptionMode == InterruptionMode.DO_NOT_MIX)
         setOnAudioFocusChangeListener(audioFocusChangeListener)
         build()
       }
@@ -169,7 +171,7 @@ class AudioModule : Module() {
     } else {
       @Suppress("DEPRECATION")
       val requestType = if (interruptionMode == InterruptionMode.DO_NOT_MIX) {
-        AudioManager.AUDIOFOCUS_GAIN_TRANSIENT
+        AudioManager.AUDIOFOCUS_GAIN
       } else {
         AudioManager.AUDIOFOCUS_GAIN_TRANSIENT_MAY_DUCK
       }
